pkgs/applications/altcoins/wownero.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchgit, cmake, pkgconfig, git
+, boost, miniupnpc, openssl, unbound, cppzmq
+, zeromq, pcsclite, readline, libsodium
+, CoreData, IOKit, PCSC
+}:
+
+assert stdenv.isDarwin -> IOKit != null;
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "wownero-${version}";
+
+  version = "0.3.1.3";
+  src = fetchgit {
```

Comment:
please use `fetchFromGitHub` instead of `fetchgit`

Reply:
Thanks @kalbasit, i needed the `fetchSubmodules = true;` part from fetchgit, is it support in fetchFromGithub now?

Reply:
Yes it's supported, see the source for fetchFromGitHub [here](https://github.com/NixOS/nixpkgs/blob/05a75a40e6954a4657be353b8358ba4e3686dbf1/pkgs/top-level/all-packages.nix#L260-L291)

Reply:
Thanks! updated.